mock-kubernetes-client/client/client.go

```diff
@@ -660,6 +661,7 @@ func (client *MockClient) Status() ctrlClient.StatusWriter {
 }
 
 func (client *MockClient) createEvent(event *corev1.Event) {
+	log.Log.Info("Creating event", "name", event.Name)
```

Comment:
That should be a debug statement not an info statement. I would prefer if we also log the corresponding namespace and cluster name.